com.civicrm.hrjobroles/js/src/job-roles/controllers/add-new-role.html

```diff
@@ -11,7 +11,7 @@
       </div>
     </header>
   </div>
-  <div class="panel panel-default panel-nested">
+  <span class="panel panel-default panel-nested">
```

Comment:
why changed to span?

Reply:
That must have been a mistake. I will revert. Thanks for catching that 👍